### ISSUE TITLE
[server] Upgrade to relay-local-schema > 0.5.3

### ISSUE THREAD

**Issue by thelordoftheboards** [codefoundries/UniversalRelayBoilerplate#217]
Version 0.5.3 causes:

```
<project root>/node_modules/relay-local-schema/lib/NetworkLayer.js:49
    var _ref2 = _asyncToGenerator(regeneratorRuntime.mark(function _callee(requestType, request) {
                                  ^

ReferenceError: regeneratorRuntime is not defined
    at <project root>/node_modules/relay-local-schema/lib/NetworkLayer.js:49:35
    at <project root>/node_modules/relay-local-schema/lib/NetworkLayer.js:93:4
    at Object.<anonymous> (<project root>/node_modules/relay-local-schema/lib/NetworkLayer.js:100:2)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
```

**Comment by thelordoftheboards** [comment]
Seems like it's fixed in 0.5.4

**Comment by thelordoftheboards** [comment]
Ops commit did not actually include it.
